code/game/machinery/computer/abnormality_archive.dm

```diff
@@ -0,0 +1,132 @@
+			record_data["line9"] = remove_paper_commands(popleft(note_list))
+			record_data["line10"] = remove_paper_commands(popleft(note_list))
+			record_data["line11"] = remove_paper_commands(popleft(note_list))
+			record_data["line12"] = remove_paper_commands(popleft(note_list))
+			record_data["line13"] = remove_paper_commands(popleft(note_list))
+			record_data["line14"] = remove_paper_commands(popleft(note_list))
+			record_data["line15"] = remove_paper_commands(popleft(note_list))
+			record_data["line16"] = remove_paper_commands(popleft(note_list))
+			record_data["line17"] = remove_paper_commands(popleft(note_list))
+			record_data["line18"] = remove_paper_commands(popleft(note_list))
+			record_data["line19"] = remove_paper_commands(popleft(note_list))
+			record_data["line20"] = remove_paper_commands(popleft(note_list))
+			record_data["line21"] = remove_paper_commands(popleft(note_list))
+			record_data["line22"] = remove_paper_commands(popleft(note_list))
+			record_data["line23"] = remove_paper_commands(popleft(note_list))
+			record_data["line24"] = remove_paper_commands(popleft(note_list))
+			record_data["line25"] = remove_paper_commands(popleft(note_list))
+			record_data["line26"] = remove_paper_commands(popleft(note_list))
+			record_data["line27"] = remove_paper_commands(popleft(note_list))
+			record_data["line28"] = remove_paper_commands(popleft(note_list))
+			record_data["line29"] = remove_paper_commands(popleft(note_list))
+			record_data["line30"] = remove_paper_commands(popleft(note_list))
+			record_data["line31"] = remove_paper_commands(popleft(note_list))
+			record_data["line32"] = remove_paper_commands(popleft(note_list))
+			record_data["line33"] = remove_paper_commands(popleft(note_list))
+			record_data["line34"] = remove_paper_commands(popleft(note_list))
+			record_data["line35"] = remove_paper_commands(popleft(note_list))
+			record_data["line36"] = remove_paper_commands(popleft(note_list))
+			record_data["line37"] = remove_paper_commands(popleft(note_list))
+			record_data["line38"] = remove_paper_commands(popleft(note_list))
+			record_data["line39"] = remove_paper_commands(popleft(note_list))
+			record_data["line40"] = remove_paper_commands(popleft(note_list))
+			record_data["line41"] = remove_paper_commands(popleft(note_list))
+			record_data["line42"] = remove_paper_commands(popleft(note_list))
+			record_data["line43"] = remove_paper_commands(popleft(note_list))
+			record_data["line44"] = remove_paper_commands(popleft(note_list))
+			record_data["line45"] = remove_paper_commands(popleft(note_list))
+			record_data["line46"] = remove_paper_commands(popleft(note_list))
+			record_data["line47"] = remove_paper_commands(popleft(note_list))
+			record_data["line48"] = remove_paper_commands(popleft(note_list))
+			record_data["line49"] = remove_paper_commands(popleft(note_list))
+			record_data["line50"] = remove_paper_commands(popleft(note_list))
```

Comment:
This is very not good code
Consider assembling a single list with multiple data types and passing that as the UI data instead of exactly 50 sets of data

Reply:
I want you to know. Very specifically. That only 10% of this is my own code. Im piggybacking off of the fish catalog Js. I dont understand Javascript. When i look up commands or helpful idenfifiers i get nothing. All i know is that the variables go in as a list.

Reply:
I honestly shouldnt of made this or touched Javascript. I just thought it would be nice to do.